src/components/ListDetails/ListDetails.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import * as S from './Style'
+import { serviceNode } from '../../services/node/node'
+
+import ItemDetails from '../ItemDetails/ItemDetails'
+import Loader from '../Loader/Loader'
+import Status from '../Status/Status'
+
+const ListDetails = ({ }) => {
+  const [expenseList, setExpenseList] = useState(undefined)
```

Comment:
A jesli wywolasz useState() po prostu to nie to samo?

Reply:
nie wiem, ale sprawdze